connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewClient.kt

```diff
@@ -39,6 +43,29 @@ internal class StripeConnectWebViewClient(
     }
 ) : WebViewClient() {
 
+    override fun shouldOverrideUrlLoading(view: WebView?, request: WebResourceRequest?): Boolean {
+        view ?: return false // the view shouldn't be null, but if it is then don't handle the request
+        val url = request?.url ?: return false // if the request isn't for a url, then don't handle it
+        return if (url.host in ALLOWLISTED_HOSTS) {
+            logger.warning("(StripeConnectWebViewClient) Received pop-up for allow-listed host: $url")
```

Comment:
Is this a warning because we don't expect getting any of these given that messaging between windows on Android isn't supported? If this is the case, we should add a TODO here to add an analytic so we can setup alerts for it.

Reply:
I was going off of this in the spec: `If host is allowlisted (connect.stripe.com or connect-js.stripe.com), print warning (stub out todo to log analytic error) because this isn't supported`

Is that still accurate @mludowise-stripe? 👍 on adding an analytic here

Reply:
🤦‍♀️ I forgot I wrote that in the jira ticket. Yes, it's because we can't currently handle JS messaging between popup windows on Android so we just want to log if we somehow get into that state and setup alerting for it.